docs/src/main/sphinx/glossary.rst

```diff
@@ -0,0 +1,202 @@
+========
+Glossary
+========
+
+The glossary contains a list of key Trino terms and definitions.
+
+Terms A-E
+---------
+
+.. _glossCatalog:
+
+Catalog
+    Catalogs define and name a configuration for connecting to a data source,
+    allowing users to query the connected data. Each catalog's configuration
+    specifies a :ref:`connector <glossConnector>` to define which data source
+    the catalog connects to. For more information, see
+    :ref:`trino-concept-catalog`.
+
+.. _glossCA:
+
+Certificate Authority (CA)
+    A trusted organization that examines and validates organizations and their
+    proposed server URIs, and issues digital certificates verified as valid for
+    the requesting organization.
+
+.. _glossCert:
+
+Certificate
+    A public key `certificate
+    <https://en.wikipedia.org/wiki/Public_key_certificate>`_ issued by a CA,
+    sometimes abbreviated as cert, that verifies the ownership of a server's
+    keys. Certificate format is specified in the `X.509
+    <https://en.wikipedia.org/wiki/X.509>`_ standard.
+
+Cluster
+    A Trino cluster provides the resources to run queries against numerous data
+    sources. Clusters define the number of workers, the configuration for the
```

Comment:
A cluster is one or more nodes, that are divided into one coordinator and zero or more workers. Since there's no `Coordinator` term, I suggest to only change the wording here:
```suggestion
    sources. Clusters define the number of nodes, the configuration for the
```

Reply:
Should this be even included here, if it's already defined in the Concepts section? There should be a single source of truth for terms like this.

Reply:
I want the glossary to be the single source of truth, and the concepts to be a lighter and more informal way to get started without having to see every possible term all at once.

Reply:
But will also ask @electrum about this point as well to get his input.